uniffi_bindgen/src/bindings/kotlin/templates/wrapper.kt

```diff
@@ -57,4 +60,9 @@ import java.util.concurrent.atomic.AtomicLong
 {% include "ObjectTemplate.kt" %}
 {% endfor %}
 
+// Callback Interfaces
+{% for obj in ci.iter_callback_interface_definitions() %}
```

Comment:
nano-nit: consider using a different name for the loop index variable here, e.g. `cb` or `callback` or something.